src/bsoncxx/builder/stream/value_context.hpp

```diff
@@ -60,8 +60,11 @@ class value_context {
 
     operator single_context();
 
+#if !defined(_MSC_VER)
```

Comment:
If you want to check the exact version of the compiler, use: `#if defined(_MSC_VER) && _MSC_VER > 1900023506`